web/src/main.js

```diff
@@ -15,6 +17,11 @@ Vue.config.productionTip = false;
 
 ApiService.init();
 
+SentryInit({
+  dsn: 'https://9c7281f0dfc5402a953698256bd067e0@sentry.io/1437129',
+  integrations: [new SentryVue({ Vue })],
+});
```

Comment:
Might want to only do this in the production environment.

Reply:
I believe the library handles this for you.... let me check.

Reply:
I got a notification, but maybe you were testing in production mode.

Reply:
Nope, you're right.  Need to check myself.